opus/application/static_media/js/browse.js

```diff
@@ -424,6 +445,22 @@ var o_browse = {
         });
     }, // end browse behaviors
 
+    // update order arrows right away when user clicks on sorting arrows in pill or table header
+    // sync up arrows in both sorting pill and table header
+    updateOrderIndicator: function(headerOrderIndicator, pillOrderIndicator, isDescending) {
```

Comment:
I don't really understand how this works currently. It assumes that there is always the same column in the sort order pills and the table header, so all the input parameters are valid and you can swap both. But it's entirely possible to be sorting on some column that is not currently in the table (e.g. sort on a column, then select metadata and delete that column). In this case I assume some of these variables are invalid so I don't know what it does then.